src/index.html

```diff
@@ -2,12 +2,15 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Dosti App</title>
+  <title>Dosti</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 </head>
-<body>
+<body class="mat-typography">
```

Comment:
suggestion: instead of adding stylings directly to the index.html, lets keep the responsibility in the application. add it to the root element within that instead.

Reply:
Removed

<body>

Reply:
Robbie,
When I remove "class="mat-typography"", it was spoiling the Looks and Feel on the Home Page. Looks like I need your help on this.

Reply:
![image](https://user-images.githubusercontent.com/8255269/213921811-12d37c37-5896-4acb-b11d-2f50284a4816.png)